Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -1086,7 +1086,8 @@ def main():
         'app': demisto.params().get('app', '-'),
         'username': demisto.params()['authentication']['identifier'],
         'password': demisto.params()['authentication']['password'],
-        'verify': VERIFY_CERTIFICATE
+        'verify': VERIFY_CERTIFICATE,
+        'basic': True
```

Comment:
If you set it to basic at all cases we wouldn't be able to use `auth token`:

<img src=https://user-images.githubusercontent.com/53565845/111486910-91e08800-8740-11eb-881c-5ec74ba0fd2a.png width=500>

In what cases do we need a basic auth?